bzt/modules/ab.py

```diff
@@ -68,7 +68,7 @@ def get_widget(self):
         return self.widget
 
     def __first_http_request(self):
-        for request in self.scenario.get_requests():
+        for request in self.get_requests():
```

Comment:
So now there're two ways to get requests: `ScenarioExecutor.get_requests()` and `Scenario.get_requests()`. To avoid confusion we should probably make `Scenario.get_requests()` method private, or hide it in some other way.